preview_generator/preview/builder/image__imconvert.py

```diff
@@ -76,11 +76,16 @@ def __load_mimetypes(cls) -> typing.List[str]:
             # (need to remove the mimetype on Ubuntu but useless on Debian
             mimes.remove("image/svg+xml")
 
-        # HACK - G.M - 2019-10-31 - Handle raw format only if ufraw-batch is installed as most common
-        # default imagemagick configuration delegate raw format to ufraw-batch.
+        # HACK - G.M - 2019-10-31 - Handle raw format only if ufraw-batch is installed
+        # as most common default imagemagick configuration delegate raw format to ufraw-batch.
         if executable_is_available("ufraw-batch"):
-            for mimetype in cls.SUPPORTED_RAW_CAMERA_MIMETYPE:
-                mimes.append(mimetype)
+            for mimetype_mapping in cls.SUPPORTED_RAW_CAMERA_MIMETYPE_MAPPING:
+                mimes.append(mimetype_mapping[0])
```

Comment:
Related to Damien comment, this code `mimetype_mapping[0]`  is not very readable.

Reply:
I see `MimetypeMapping` class. Can you use it here (in `cls.SUPPORTED_RAW_CAMERA_MIMETYPE_MAPPING`) ?

Reply:
done.